alliance/config/settings/base.py

```diff
@@ -92,11 +111,12 @@
 THIRD_PARTY_APPS = (
 )
 
-#PROJECT_APPS = (
-#    'apps.shared',
-#    'apps.backlog',
-#    'apps.accounts',
-#)
+PROJECT_APPS = (
+    'alliance.core',
+    'alliance.apps.backlog',
+    'alliance.apps.accounts',
+    'apps.shared'
```

Comment:
I know this seems odd, but it is what is working on heroku. "alliance.apps.shared" caused import errors.

Just for my own sanity, I viewed the file as it is deployed on the server:

![screenshot from 2015-10-28 14 24 03](https://cloud.githubusercontent.com/assets/1907202/10800427/c1f4664a-7d7f-11e5-94cf-e34073b5fd0e.png)

![screenshot from 2015-10-28 14 22 42](https://cloud.githubusercontent.com/assets/1907202/10800432/c698acc4-7d7f-11e5-8894-c50517b78a79.png)